sdcard/rootfs/kube-systemd/etc/kubernetes/dynamic-env/os/odrobian.sh

```diff
@@ -0,0 +1,58 @@
+DOCKER_HYPRIOT_DEB="docker-hypriot_1.10.3-1_armhf.deb"
+DOCKER_HYPRIOT_DEB_FILE="/tmp/"$DOCKER_HYPRIOT_DEB
+
+os_install(){
+
+	os_upgrade
+
+	if [[ ! -f $(which curl 2>&1) ]]; then
+		echo "Installing curl ..."
+		apt-get install curl -y
+	fi
+
+	if [[ ! -f $(which docker 2>&1) ]]; then
+		echo "Installing docker ..."
+		install_docker
+	fi
+
+	if [[ ! -f $(which sudo 2>&1) ]]; then
+		echo "Installing sudo ..."
+		apt-get install sudo -y
+	fi
+
+	echo "Installing dbus and avahi-daemon ..."
+	apt-get install -y dbus avahi-daemon
+}
+
+install_docker(){
+	apt-get install lxc lvm2 cgroup-tools apparmor bridge-utils -y
```

Comment:
I think lxc isn't required here. not sure about lvm2 and cgroup-tools when you're using a statically linked daemon